pkg/validate/image.go

```diff
@@ -198,7 +198,7 @@ func testPullPublicImage(c internalapi.ImageManagerService, imageName string, po
 
 // pullImageList pulls the images listed in the imageList.
 func pullImageList(c internalapi.ImageManagerService, imageList []string, podConfig *runtimeapi.PodSandboxConfig) []string {
-	var ids []string
+	ids := []string{}
```

Comment:
The issue was that the previous pre-allocation to `len(imageList)` compromised the test result. So this is fixed now. :)

Reply:
thanks for the fix